### PR TITLE
feat: Added messageStackDirection in Constant

### PR DESCRIPTION
**Pull request by chohongm**
## Changes
- 

ticket: [AC-4550]

## Additional Notes
- Added `messageStackDirection` in `Constant`

## Checklist
Before requesting a code review, please check the following:
- [ ] **[Required]** CI has passed all checks.
- [ ] **[Required]** A self-review has been conducted to ensure there are no minor mistakes.
- [ ] **[Required]** Unnecessary comments/debugging code have been removed.
- [ ] **[Required]** All requirements specified in the ticket have been accurately implemented.
- [ ] Ensure the ticket has been updated with the sprint, status, and story points.

[AC-4550]: https://sendbird.atlassian.net/browse/AC-4550?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/const.ts [hunk 1]
```diff
@@ -187,6 +187,11 @@ export interface Constant extends ConstantFeatureFlags, ConstantAIFeatures, Cons
    * @description Message input state controlling properties.
    */
   messageInputControls?: MessageInputControls;
+  /**
+   * @public
+   * @description
```
Comment (bang9) [inline src/const.ts]: 추가할 거 없으면 description 빼도 될거같아용

Comment (bang9) [reply src/const.ts]: Readme.md 테이블에도 추가 하나 해주실 수 있을까요?

Comment (chohongm) [reply src/const.ts]: 넵!

File: README.md [hunk 2]
```diff
@@ -108,25 +108,26 @@ export default App;
...
 | `dateLocale` | `Locale` | No | `enUS` | Locale value to be applied to string values of message timestamp and date separator. Locale values must be imported from `date-fns`.                                                    |
-| `enableHideWidgetForDeactivatedUser` | `boolean`  | No       | `false`       | Determines whether the chatbot widget is hidden when the user is deactivated.                                                                                                           |
+| `enableHideWidgetForDeactivatedUser` | `boolean` | No       | `false`       | Determines whether the chatbot widget is hidden when the user is deactivated.                                                                                                           |
 | `enableWidgetExpandButton` | `boolean` | No | `false` | Determines whether the expand button is displayed in the chatbot widget. |
+| `messageStackDirection` | `'top' \| 'bottom'` | No | N/A | Determines direction at which message stack starts in the message list. |
```
Comment (bang9) [inline README.md]: 기본값 bottom 으로 넣는건 어떠신가용

File: README.md [hunk 3]
```diff
@@ -108,25 +108,26 @@ export default App;
...
 | `dateLocale` | `Locale` | No | `enUS` | Locale value to be applied to string values of message timestamp and date separator. Locale values must be imported from `date-fns`.                                                    |
-| `enableHideWidgetForDeactivatedUser` | `boolean`  | No       | `false`       | Determines whether the chatbot widget is hidden when the user is deactivated.                                                                                                           |
+| `enableHideWidgetForDeactivatedUser` | `boolean` | No       | `false`    | Determines whether the chatbot widget is hidden when the user is deactivated.                                                                                                           |
 | `enableWidgetExpandButton` | `boolean` | No | `false` | Determines whether the expand button is displayed in the chatbot widget. |
+| `messageStackDirection` | `'top' \| 'bottom'` | No | `bottom` | Determines direction at which message stack starts in the message list. |
```
Comment (bang9) [inline README.md]:
```suggestion
| `messageStackDirection` | `'top' \| 'bottom'` | No | `'bottom'` | Determines direction at which message stack starts in the message list. |
```

**Review by bang9 — APPROVED** [review]

**Pull request merged**